ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,16 @@
+# source:
+# https://raw.githubusercontent.com/pulp/crane/3c657fe53020b804ce60d907a82b4fb19aebe532/deployment/apache22.conf
+# Use this config with Apache 2.2 or earlier
+Listen 5000
+<VirtualHost *:5000>
+    WSGIScriptAlias / /usr/share/crane/crane_el6.wsgi
+    <Location /crane>
+        Order Deny,Allow
+        Deny from all
+        Allow from localhost
```

Comment:
For testing purposes, I believe connections to Crane should be allowed from clients other than localhost.